aequilibrae/distribution/gravity_application.py

```diff
@@ -188,17 +195,17 @@ def apply(self):
 
         # We use the model application parameters in case they were provided
         # not the standard way of using this tool)
-        for p in ipf.parameters:
+        for p in self.__ipf.parameters:
             if p in self.parameters:
-                ipf.parameters[p] = self.parameters[p]
+                self.__ipf.parameters[p] = self.parameters[p]
 
-        # apply fratar
-        ipf.fit()
-        self.output = ipf.output
-        self.gap = ipf.gap
+        # apply Fratar
+        self.__ipf.fit()
+        self.output = self.__ipf.output
+        self.gap = self.__ipf.gap
 
-        q = ipf.report.pop(0)
-        for q in ipf.report:
+        _ = self.__ipf.report.pop(0)
```

Comment:
what does it do? what is there that must be removed?

Reply:
Appends IPF report to the distribution report. Very old code, so it is just a bit dump.  Just replaced it